src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ArgCoercion.txt

```diff
@@ -55,4 +55,4 @@ Error({Kind:ErrorKind.InvalidArgument})
 3
 
 >> Last([ {a:2}, {a:true, b:true, c:20}, {a:2, b:2, c:true} ]).c
```

Comment:
>Last([ {a:2}, {a:true, b:true, c:20}, {a:2, b:2, c:true} ]).c [](http://example.com/codeflow?start=3&length=61)

This is a breaking change, so this should only be done with the new PowerFxV1CompatibilityRules feature enabled.